### PR TITLE
Create events/appointments in Calendar

### PR DESCRIPTION
**Pull request by jacqueline8711**
## Issue

This fixes the following issue(s): #410 + #635 

## Screenshots
screenshot_20180307-130808
screenshot_20180307-130943
screenshot_20180307-125907

### CODE REVIEW

File: app/src/main/java/de/tum/in/tumcampusapp/component/tumui/calendar/CalendarActivity.java [hunk 1]
```diff
@@ -183,6 +183,10 @@ public boolean onOptionsItemSelected(MenuItem item) {
         } else if (i == R.id.action_delete_calendar) {
             deleteCalendarFromGoogle();
             return true;
+        } else if (i == R.id.action_create_event){
+            //new CreateEventDialogFragment().show(getSupportFragmentManager(), "CreateEventDialog");
```
Comment (pfent) [inline app/src/main/java/de/tum/in/tumcampusapp/component/tumui/calendar/CalendarActivity.java]: please don't leave commented code

File: app/src/main/java/de/tum/in/tumcampusapp/component/tumui/calendar/CreateEventActivity.java [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+
+import de.tum.in.tumcampusapp.R;
+import de.tum.in.tumcampusapp.api.tumonline.TUMOnlineConst;
+import de.tum.in.tumcampusapp.component.other.generic.activity.ActivityForAccessingTumOnline;
+import de.tum.in.tumcampusapp.component.tumui.calendar.model.CreateEvent;
+import de.tum.in.tumcampusapp.utils.DateUtils;
+import de.tum.in.tumcampusapp.utils.Utils;
+
+public class CreateEventActivity extends ActivityForAccessingTumOnline<CreateEvent> {
+    Calendar start, end;
```
Comment (pfent) [inline app/src/main/java/de/tum/in/tumcampusapp/component/tumui/calendar/CreateEventActivity.java]: probably make those private

File: app/src/main/java/de/tum/in/tumcampusapp/component/tumui/calendar/CalendarActivity.java (hunk 1 above)
Comment (jacqueline8711) [reply app/src/main/java/de/tum/in/tumcampusapp/component/tumui/calendar/CalendarActivity.java]: @pfent I know, could you please ignore this pull request for now, I know there are lots off errors and stuff in here right now. I just haven't pushed the next commit yet.

Comment (pfent) [reply app/src/main/java/de/tum/in/tumcampusapp/component/tumui/calendar/CalendarActivity.java]: sure, no worriez.
didn't notice the WIP label, my bad

File: app/src/main/AndroidManifest.xml [hunk 3]
```diff
@@ -685,13 +686,11 @@
                 <action android:name="com.google.firebase.INSTANCE_ID_EVENT"/>
             </intent-filter>
         </service>
-
         <service
```
[the remark this answers is not in the document]
Comment (jacqueline8711) [reply app/src/main/AndroidManifest.xml]: the TCA-Bot does not seem to recognise Kotlin classes...

File: app/src/main/java/de/tum/in/tumcampusapp/component/tumui/calendar/CalendarDao.java [hunk 4]
```diff
@@ -14,8 +14,8 @@
     @Query("SELECT c.* FROM calendar c WHERE status != 'CANCEL'")
     List<CalendarItem> getAllNotCancelled();
 
-    @Query("SELECT c.* FROM calendar c WHERE dtstart LIKE '%' || :date || '%' AND status != 'CANCEL' ORDER BY dtstart ASC")
-    List<CalendarItem> getAllByDateNotCancelled(String date);
+    @Query("SELECT c.* FROM calendar c WHERE dtstart LIKE '%' || :date || '%' ORDER BY dtstart ASC")
+    List<CalendarItem> getAllByDate(String date);
```
Comment (pfent) [inline app/src/main/java/de/tum/in/tumcampusapp/component/tumui/calendar/CalendarDao.java]: Silently swallowing events wasn't a great idea in the first place :see_no_evil:. Thanks for fixing #635

File: app/src/main/java/de/tum/in/tumcampusapp/component/tumui/calendar/model/CalendarItem.kt [hunk 5]
```diff
@@ -30,12 +30,14 @@ data class CalendarItem(@PrimaryKey
...
+        return if(status.equals("CANCEL")){
+            IntegratedCalendarEvent.getDisplayColorFromColor(0xd50000)
+        } else if (title.endsWith("VO") || title.endsWith("VU")) {
+            IntegratedCalendarEvent.getDisplayColorFromColor(0x86c415)
         } else if (title.endsWith("UE")) {
-            IntegratedCalendarEvent.getDisplayColorFromColor(-0x8000)
+            IntegratedCalendarEvent.getDisplayColorFromColor(0xfcad18)
         } else {
-            IntegratedCalendarEvent.getDisplayColorFromColor(-0xffff01)
+            IntegratedCalendarEvent.getDisplayColorFromColor(0x5598D3)
```
Comment (pfent) [inline app/src/main/java/de/tum/in/tumcampusapp/component/tumui/calendar/model/CalendarItem.kt]: those colors should probably move to `colors.xml`

**Pull request merged**